scipy/_lib/tests/test_public_api.py

```diff
@@ -317,7 +317,12 @@ def check_importable(module_name):
 
 
 @pytest.mark.parametrize(("module_name", "correct_module"),
-                         [('scipy.optimize.cobyla', None),
+                         [('scipy.integrate.odepack', None),
+                          ('scipy.integrate.quadpack', None),
+                          ('scipy.integrate.dop', None),
+                          ('scipy.integrate.lsoda', None),
+                          ('scipy.integrate.vode', None),
```

Comment:
I think it would be easier to keep this sorted alphabetically like the others.
```suggestion
                         [('scipy.integrate.dop', None),
                          ('scipy.integrate.lsoda', None),
                          ('scipy.integrate.odepack', None),
                          ('scipy.integrate.quadpack', None),
                          ('scipy.integrate.vode', None),
```

Reply:
If it's ok I would actually like to keep the present order as I have been copying these from the `PRIVATE_BUT_PRESENT_MODULES` list on line 77 so keeping them in the same order makes it easier to cross reference

Reply:
Can we just also sort `PRIVATE_BUT_PRESENT_MODULES` please? It's already _very_ close to sorted. To double-check, I just did
```
q = """<content of PRIVATE_BUT_PRESENT_MODULES>"""
print("\n".join(sorted(q.splitlines())))
```
and copied the result back; the required changes are very small
```diff
@@ -82,10 +82,10 @@ PRIVATE_BUT_PRESENT_MODULES = [
     'scipy.fftpack.helper',
     'scipy.fftpack.helper',
     'scipy.fftpack.pseudo_diffs',
     'scipy.fftpack.realtransforms',
-    'scipy.integrate.odepack',
-    'scipy.integrate.quadpack',
     'scipy.integrate.dop',
     'scipy.integrate.lsoda',
+    'scipy.integrate.odepack',
+    'scipy.integrate.quadpack',
     'scipy.integrate.vode',
     'scipy.interpolate.dfitpack',
     'scipy.interpolate.fitpack',
@@ -98,8 +98,6 @@ PRIVATE_BUT_PRESENT_MODULES = [
     'scipy.io.arff.arffread',
     'scipy.io.harwell_boeing',
     'scipy.io.idl',
-    'scipy.io.mmio',
-    'scipy.io.netcdf',
     'scipy.io.matlab.byteordercodes',
     'scipy.io.matlab.mio',
     'scipy.io.matlab.mio4',
@@ -109,6 +107,8 @@ PRIVATE_BUT_PRESENT_MODULES = [
     'scipy.io.matlab.mio_utils',
     'scipy.io.matlab.miobase',
     'scipy.io.matlab.streams',
+    'scipy.io.mmio',
+    'scipy.io.netcdf',
     'scipy.linalg.basic',
     'scipy.linalg.decomp',
     'scipy.linalg.decomp_cholesky',
```

Reply:
Updated

Reply:
Thanks! :)